src/main/java/org/folio/circulation/resources/CirculationRulesResource.java

```diff
@@ -115,14 +145,91 @@ private void put(RoutingContext routingContext) {
     LoanCirculationRulesEngineResource.clearCache(new WebContext(routingContext).getTenantId());
     RequestCirculationRulesEngineResource.clearCache(new WebContext(routingContext).getTenantId());
 
-    loansRulesClient.put(rulesInput.copy()).thenAccept(result ->
-      result.applySideEffect(response -> {
+    clients.circulationRulesStorage().put(rulesInput.copy())
+      .thenAccept(res -> res.applySideEffect(response -> {
         if (response.getStatusCode() == 204) {
           SuccessResponse.noContent(routingContext.response());
         } else {
           ForwardResponse.forward(routingContext.response(), response);
         }
-    }, cause -> cause.writeTo(routingContext.response())));
+      }, cause -> cause.writeTo(routingContext.response())));
+  }
+
+  private void validatePolicy(Map<String, Set<String>> existingPoliciesIds, String policyType, List<CirculationRulesParser.PolicyContext> policies, Token token) {
```

Comment:
Please split these parameters according to the standard maximum line length for FOLIO (80 characters, although 100 is generally accepted)

Reply:
done